src/vector/v.surf.icw/v.surf.icw.py

```diff
@@ -112,11 +92,8 @@
 # % key: r
 # % description: Use (d^n)*log(d) instead of 1/(d^n) for radial basis function
 # %end
-# %option
-# % key: workers
-# % type: integer
-# % options: 1-256
-# % answer: 1
+# %option G_OPT_M_NPROCS
+# % options: 1-1024
```

Comment:
No need to limit the values. Definitively not the upper limit. `1-` should work just fine.

There was some work done for C and OpenMP (not Python yet) to use zero and negative in a useful way.